packages/renderer/MdToHtml/linkReplacement.ts

```diff
@@ -120,9 +119,11 @@ export default function(href: string, options: Options = null): LinkReplacementR
 		resourceFullPath = url;
 	} else if (options.plainResourceRendering || options.linkRenderingType === 2) {
 		icon = '';
+		if (title) attrHtml.push(`title='Ctrl + Click to open ${htmlentities(title)}'`);
 		attrHtml.push(`href='${htmlentities(href)}'`);
 	} else {
 		attrHtml.push(`href='${htmlentities(hrefAttr)}'`);
+		if (title) attrHtml.push(`title='${htmlentities(title)}'`);
 		if (js) attrHtml.push(js);
```

Comment:
This MdToHtml is used in many places, including the viewer where `Ctrl + Click` is not needed. Whatever the fix is, it should be scoped to just the TinyMCE editor

Reply:
I found another [solution here](https://www.tiny.cloud/docs/plugins/opensource/link/#link_context_toolbar) that will be less intrusive. 
Looks like this: 
![image](https://github.com/laurent22/joplin/assets/59203815/132fc934-9455-404d-9f1a-235bed33d244)


It has an icon at the end to visit the link from the editor